.github/workflows/enteprise.yml

```diff
@@ -32,7 +32,8 @@ jobs:
           python-version-file: '.python-version'
           cache: "pip"
 
-      - run: pip install -r requirements.txt
+      - run: pip install --user pipenv
+      - run: pipenv install
```

Comment:
if you feel adventurous, you can try making a new composite github action that includes those two run commands and the previous setup python, like we already did for ./.github/actions/galaxy below

in this way you don't duplicate commands in all the workflows